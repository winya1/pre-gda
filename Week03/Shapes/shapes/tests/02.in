20
21 -79 100 -84 87 -75
14 -24 -4 -70 -74 38
-3 -62 76 77 33 6
33 -11 -14 -95 -68 -36
54 44 5 30 -44 79
-18 54 -41 -86 10 -87
15 10 95 -83 41 80
34 52 76 52 -91 69
-73 92 -64 82 47 10
-27 93 -77 100 8 82
33 77 23 -65 15 -99
89 -10 -83 14 56 -59
-59 -1 61 43 66 -40
-95 -93 95 60 69 12
29 18 -41 33 -21 -20
-14 -50 -66 -66 81 -12
7 -46 -50 -73 99 -94
18 56 68 11 -82 -60
52 -16 63 -72 -13 -98
60 62 -87 83 90 95
