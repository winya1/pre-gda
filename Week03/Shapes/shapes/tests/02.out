Case #1: 323.00 161.09 711.00 176.00
Case #2: 2582.00 285.75 9504.00 392.00
Case #3: 184.00 319.83 10981.00 436.00
Case #4: 3654.50 280.28 8484.00 370.00
Case #5: 1543.50 224.32 4802.00 294.00
Case #6: 3581.50 336.64 7191.00 384.00
Case #7: 4009.00 369.06 13040.00 486.00
Case #8: 357.00 336.01 2839.00 368.00
Case #9: 231.00 291.10 9840.00 404.00
Case #10: 152.50 174.06 1530.00 206.00
Case #11: 398.00 354.20 3168.00 388.00
Case #12: 4610.00 389.75 12556.00 490.00
Case #13: 5090.00 341.91 10375.00 416.00
Case #14: 2571.00 493.27 29070.00 686.00
Case #15: 1705.00 191.04 3710.00 246.00
Case #16: 228.00 313.33 7938.00 402.00
Case #17: 2610.00 317.31 7152.00 394.00
Case #18: 5150.00 386.38 17400.00 532.00
Case #19: 2271.00 242.03 6232.00 316.00
Case #20: 2740.50 370.50 5841.00 420.00
